app/lib/three_scale/jobs.rb

```diff
@@ -0,0 +1,102 @@
+# Periodic tasks code are extracted to constants so that
+# it can be easily run by tests.
+#
+module ThreeScale
+  module Jobs
+
+    class Task
+      def initialize(object, method, *args)
+        @object, @method, @arguments = object, method, args
+      end
+
+      def name
+        [@object, @method, @arguments].join('_').tr('():.', '_')
```

Comment:
I'm not sure this is really needed but to distinguish between dots and the 3 strings concatenation, perhaps it can be `join("__")`.

Reply:
jut as a quick visual clue in case anybody has any chance to be confused, perhaps it will be apparent anyway.

Reply:
It is just cosmetic ... extracted from the test and also I should probably remove the trailing `_` but not worth it.